File: src/main/Python/task_3/cube.py
from i_shape_2D import IShape2D
from i_shape_3D import IShape3D


class Cube(IShape2D, IShape3D):
    def __init__(self, edge):
        self.edge = edge

    def area(self):
        return 6 * self.edge * self.edge

    def volume(self):
        return self.edge * self.edge * self.edge

File: src/main/java/task_3/Cube.java
package task_3;

import task_3.Interfaces.Shape2D;
import task_3.Interfaces.Shape3D;

public class Cube implements Shape2D, Shape3D {
    private int edge;
    public Cube(int edge) {
        this.edge = edge;
    }
    @Override
    public double area() {
        return 6 * edge * edge;
    }
    @Override
    public double volume() {
        return edge * edge * edge;
    }
}
